### DISCUSSION TITLE
cursor and take option in t.crud

### DISCUSSION THREAD

**Discussion by aakashkag** [prisma/prisma#2931] [Q&A] [answered]
Hi,
I can see on 2.0.0-beta.7 [https://github.com/prisma/prisma/releases/tag/2.0.0-beta.7](https://github.com/prisma/prisma/releases/tag/2.0.0-beta.7) version Prisma team removed  first, last, before, after arguments and introduced cursor and take. 

I am still not getting option of take and skip in t.crud. Even though i am using latest version.

 "@nexus/schema": "0.14.0",
 "nexus-plugin-prisma": "^0.16.1"

Thanks :)

**Comment by ryands17** [comment, marked as answer]
Hey @aakashkag :wave: 
You need to add this enable skip:

```ts
const nexusPrisma = nexusPrismaPlugin({
  experimentalCRUD: true,
  paginationStrategy: 'prisma',
  prismaClient: (ctx: Context) => ctx.prisma,
})
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
